applications/vasp/Performance Study 2/benchmarks/README.md

```diff
@@ -0,0 +1,12 @@
+# VASP Benchmarking
+
+talk about goals of study, different input systems, systems run on, calculation performed (energy calculation), modules and version of VASP used
+direct to VASP reframe repo in order to run your own tests using these (or other) input systems
+link to grey log to view others' VASP benchmarking results
+
+## Table of Contents
```

Comment:
Are these links supposed to link to different pages or sections?

Reply:
file shouldn't be there, deleted file